src/auth/ui/vue/types.ts

```diff
@@ -4,3 +4,13 @@
  */
 
 export type ServiceItemId = 'DOCUMENT_TYPE_SUPPORT' | 'RESOURCE_EXPLORER' | 'CODE_WHISPERER' | 'CODE_CATALYST'
```

Comment:
`ServiceItemId` as a name might have too much overlap with service ids found in AWS SDK (and the service models. 

This seems more like a "component id"?
```suggestion
export type toolkitComponent = 'doctypeTypes' | 'awsExplorer' | 'codewhisperer' | 'codecatalyst'
```